docs/book/src/clusterctl/commands/init.md

```diff
@@ -53,7 +53,12 @@ Note it is not possible to skip automatic installation of the `cluster-api` core
 
 #### Provider version
 
-The `clusterctl init` command by default installs the latest version available for each selected provider.
+The `clusterctl init` command by default installs the latest version available
+for each selected provider.
+
+This does not include pre-release versions. For example, if a provider has
```

Comment:
What about moving this in the note (or in a separated note) providing an example on how to install pre-releases?